### PR TITLE
rdxd: Drop stub and add the real thing.

### PR DESCRIPTION
**Pull request by Herrie82**
Seeing we now have a real rdxd via webOS OSE, let's use it instead of the stub provided by OWO.

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>

### CODE REVIEW

File: meta-luneos/recipes-webos/rdxd/rdxd.bb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+inherit pkgconfig
+inherit webos_machine_impl_dep
+inherit webos_systemd
+
+WEBOS_GIT_PARAM_BRANCH = "webOS-ports/webOS-OSE"
+SRC_URI = "${WEBOS_PORTS_GIT_REPO_COMPLETE}"
+
+S = "${WORKDIR}/git"
+
+SRCREV = "f159cb54d13777e835015bdbcdf5be46e58fe539"
```
Comment (shr-project) [inline meta-luneos/recipes-webos/rdxd/rdxd.bb]: set PV :)

Comment (Herrie82) [reply meta-luneos/recipes-webos/rdxd/rdxd.bb]: If I would know what to put before +git${SRCPV} that would help ;) I have no clue.... 1.0.0-3? 3.0.0-3?

Comment (shr-project) [reply meta-luneos/recipes-webos/rdxd/rdxd.bb]: That comes from WEBOS_VERSION in the recipe you're importing (or version/X.Y.Z + submissions/N tags of the source version you've imported)

**Comment by shr-project** [comment]
LGTM

**Pull request merged**